### ISSUE TITLE
Tests completely fail to run

### ISSUE THREAD

**Issue by sjclayton** [lawrence-laz/neotest-zig#17]
**Describe the bug**
Tests completely fail to run

## **Steps To Reproduce**
1. Open zig file with tests in it
2. Try to run all or any specific test

## **Current behavior**
Tests do not get run and I get this as error output in the Neotest output panel:

`/home/shaun/Projects/ziglings.org/neotest_build.zig:5:30: error: error is ignored

    original_build_file.build(b);
    ~~~~~~~~~~~~~~~~~~~~~~~~~^~~

/home/shaun/Projects/ziglings.org/neotest_build.zig:5:30: note: consider using 'try', 'catch', or 'if'
referenced by:
    runBuild__anon_8993: /home/shaun/zig/0.12.0/files/lib/std/Build.zig:2079:27
    main: /home/shaun/zig/0.12.0/files/lib/compiler/build_runner.zig:300:29
    remaining reference traces hidden; use '-freference-trace' to see all reference traces`

## **Expected behavior**
Tests run with success or an error

## **Logs**

[Log Output](https://pastebin.com/2acpuKYA)

## **Environment**
 - OS: Arch Linux
 - Zig version: 0.12.0
 - Nvim version: 0.10.0
 - Neotest version: v5.2.5
 - Neotest Zig version: HEAD

**Comment by lawrence-laz** [comment]
Interesting, ziglings build function can return an error, didn't think this was possible.
I'll try to patch this up later today.

**Comment by lawrence-laz** [comment]
Should be fixed in #18, could you check it out?

**Comment by sjclayton** [comment]
> Should be fixed in #18, could you check it out?

Appears to be working ok now with the fix.
